sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -143,6 +143,7 @@ public partial class SearchOptions
         public System.Collections.Generic.IList<string> SearchFields { get { throw null; } }
         public Azure.Search.Documents.Models.SearchMode? SearchMode { get { throw null; } set { } }
         public System.Collections.Generic.IList<string> Select { get { throw null; } }
+        public string SemanticConfiguration { get { throw null; } set { } }
```

Comment:
Should we call this `SemanticConfigurationName` so it's obvious what people need to fill this with?

Reply:
I was going to do that but then noticed we have an existing option `ScoringProfile` which is "The name of a scoring profile to evaluate match scores...".
Since no option is using the `Name` suffix, I skipped it here.

Should we use `Name` for the new option and have it as a policy going forward?

Reply:
Whenever we have these cases where people have to link different parts of the API together, I like to give them as much help as possible.  `SemanticConfigurationName` is a really strong hint that I want `SemanticConfiguration.Name` rather than some magical serialized "semantic configuration."  We probably should have done it for `ScoringProfile` as well so people didn't think they needed a well known string value like "tanimoto" or something.